curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64ddd280d69fa818ca782d50.md

```diff
@@ -0,0 +1,69 @@
+---
+id: 64ddd280d69fa818ca782d50
+title: Step 29
+challengeType: 20
+dashedName: step-29
+---
+
+# --description--
+
+As you can see, disk 1 is going back and forth every three moves. This happens because you still need to take care of movements between the other rods.
+Instead of repeating the same code you wrote during the previous few steps and changing the rods, it would be better to move that code inside a function to call in each conditional statement. Declare an empty function named `make_allowed_move` and don't forget the `pass` keyword.
```

Comment:
```suggestion
Instead of repeating the same code you wrote during the previous few steps and changing the rods, it would be better to move that code inside a function to call in each conditional statement. Declare an empty function named `make_allowed_move()` and don't forget the `pass` keyword.
```